packages/ui/src/components/settings/RenderingSettings.tsx

```diff
@@ -69,6 +69,16 @@ export function RenderingSettings() {
           />
         </Group>
       )}
+      <Group>
+        <Label>{meta.rendering.groupByScene.name}</Label>
+        <Input
+          type="checkbox"
+          checked={Boolean(state.groupByScene)}
```

Comment:
The cast is not necessary:
```suggestion
          checked={state.groupByScene}
```
The type system prevents anything other than a boolean to be set as the value of this field.
If you want to make sure that the value read from the meta file is correct, you should create a new `BoolMetaField` class and override the `parse` method.

Reply:
you're right, not sure why i added `Boolean` here. will remove. although idk about your suggestion with `BoolMetaField`. does it make sense to create it now? I initialise `groupByScene` with a boolean value in `createProjectMetadata`. Unless someone decides to manually modify meta files this seems useless.